src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -0,0 +1,44 @@
+import { incidentDate, secondaryIncidentDate } from '../../pages';
+
+import { needsToEnter781, needsToEnter781a } from '../../utils';
+
+export function formConfig781(iterations) {
+  let configObj = {};
+  for (let index = 0; index < iterations; index++) {
+    configObj = {
+      ...configObj,
+      // 781 PAGE CONFIGS GO HERE
+      ...{
+        [`incidentDate${index}`]: {
+          title: 'PTSD Incident Date',
+          path: `disabilities/ptsd-incident-date-${index}`,
+          depends: needsToEnter781,
+          uiSchema: incidentDate.uiSchema(index),
+          schema: incidentDate.schema(index),
```

Comment:
Yeah this definitely is a great idea. I didn't know if the form system would completely overwrite the nested data each time or not.

Regarding repeating the page config, some of our main concerns were:

1. Excessive duplicate code.
2. Difficulty in reading the `config.js`. This was the biggest due to the amount of code that is already in there, and we've got a few more ancillary forms to add to this. If we did just duplicate it would probably still be advisable to have it as a separate file for readability.
3. PR code lines 😄 
4. The intention here was a "quick" solution with the hopes of going back and changing this to be truly dynamic in the future .

With all that being said, the main point of this was to get feedback from you all and what is preferable. We are flexible but want to make a decision ASAP and go with it.

If you all are ok with PRs being a little "line" heavy for these with the understanding that most code is literally just duplicated then we are ok with it too!

As far as testing the "duplicate" pages would we want to repeat the tests and test against that bit of data? (IE add an incidentDate1 for first incidentDate page, add incidentDate2 for second...etc).